UDPServer/ViewModels/StringToColorConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace UDPServer.ViewModels
+{
+    class StringToColorConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return value.ToString() == "Stopped" ? new SolidColorBrush(Colors.Red) : new SolidColorBrush(Colors.Green);
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if(value is SolidColorBrush)
+            {
+                return ((SolidColorBrush)value).Color == Colors.Red ? "Stopped" : "Listening";
+            }
+            return "Stopped";
+        }
+    }
```

Comment:
This code is similar to the other one that looks at Connected/Disconnected. 

This is a good opportunity to introduce DRY (Don't Repeat Yourself) - but of course you need to balance this with speed, deadlines etc.

So you could define a base class where the strings are mapped in the child class.
public StringToColorConverter() : base("Stopped", "Listening")

Reply:
Implemented